src/core/Akka.Streams.Tests/Dsl/FlowSelectAsyncSpec.cs

```diff
@@ -30,23 +30,23 @@
 
 namespace Akka.Streams.Tests.Dsl
 {
-    public class FlowMapAsyncSpec : AkkaSpec
+    public class FlowSelectAsyncSpec : AkkaSpec
     {
         private ActorMaterializer Materializer { get; }
 
-        public FlowMapAsyncSpec(ITestOutputHelper helper) : base(helper)
+        public FlowSelectAsyncSpec(ITestOutputHelper helper) : base(helper)
         {
             Materializer = ActorMaterializer.Create(Sys);
         }
 
         [Fact]
-        public void A_Flow_with_MapAsync_must_produce_future_elements()
+        public void A_Flow_with_SelectAsync_must_produce_future_elements()
```

Comment:
future is used instead of task in all methods here, could make sense to rename it as well in the context of SelectAsync